README.md

```diff
@@ -1 +1,121 @@
-# capstone-project-tesfaye
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Tesfaye Capstone README </b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+
+# 📖 [Portfolio mobile project] <a name="about-project"></a>
+
+**[Portfolio mobile project]** is a project that describe my skills and previous work experience.
+
+## 🛠 Built With <a name="built-with">html and css</a>
+
+### Key Features <a name="key-features"></a>
+
+- **[contact form ]**
+- **[projects show cards]**
+
```

Comment:
- [ ] In your project I cannot find any video recording. 

1. You should have recorded a video presenting the features of the project.
1. The maximum length of the video should be 5 minutes.
1. The video should contain a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
1. The video should contain a demo of the project, showing the pages in different screen sizes.
1. The video should contain a highlight about a piece of the code the student finds interesting or is proud of making.